frontend/app/templates/components/transportation/tdf/modal-splits/census-tracts-map.hbs

```diff
@@ -62,32 +62,22 @@
       @sql="select * from mta_subway_stops_v0"
       @layer={{hash
         type="circle"
-        paint=(get-layer-style 'subway-stops' 'paint')
+        paint=(get-layer-style "subway-stops" "paint")
       }}
      />
     <carto-source.layer
       @id="tracts"
       @sql="select * from nyc_census_tracts_2010"
       @layer={{hash
         type="fill"
-        paint=(get-layer-style 'selectable-feature-content' 'paint')
+        paint=(get-layer-style "selectable-feature-content" "paint")
       }} as |layer|
     >
       <Mapbox::FeatureHoverer
         @map={{map}}
         @layerId={{layer.layerId}}
         @onFeatures={{setFirstHoveredFeatureId}}
-        as |hoverer|
-      >
-        {{!-- {{#if (and hoverer.point hoverer.features)}}
-          <HoverCard @point={{hoverer.point}}>
-            <Transportation::Tdf::ModalSplits::CensusTractsMap::CensusTractPopup
-              @feature={{hoverer.features.firstObject}}
-              data-test-popup='census-tract'
-            />
-          </HoverCard>
-        {{/if}} --}}
-      </Mapbox::FeatureHoverer>
+      />
```

Comment:
@godfreyyeung what's happening here?  Hard for me to tell the diff since I don't fully understand ember components/templating yet.

Reply:
Here I delete an unused (commented out) block of code that's nested under FeatureHoverer. Deleting it cleans up the code, and also lets me use the "inline form" instead of "block form" (having both an opening and closing tag) declaration of FeatureHoverer. This fixes an lint error about unnecessary block form, since FeatureHoverer wasn't wrapping anything.